### PR TITLE
change build and depends: emacs 26.3

### PR DESCRIPTION
**Pull request by a-schaefers**
## Description of package
Change from GTK to Athena toolkit (emacs is well known to not have a proper gtk implementation and the Athena toolkit is more performant)
## Checklist

- [x] Latest upstream version.
- [x] I agree to maintain this package (**required**).
- [x] I agree to notify KISS if I can no longer maintain this package (**required**).

### CODE REVIEW

**Comment by a-schaefers** [comment]
decided against the mods i had, going basically vanilla for both Emacses

**Comment by a-schaefers** [comment]
I am unhappy with imagemagick6 overall, it was my hope in the beginning it could replace need for maintaining so many image libs, but imagemagick just requires those same libs anyways. let's just use defaults for images

**Comment by dylanaraps** [comment]
Sure thing. I'd much rather we package ImageMagick (7) and drop 6 too. :+1:

File: community/emacs/depends [hunk 1]
```diff
@@ -1,2 +1,4 @@
-ncurses
 zlib
+ncurses
+libxaw3d
+imagemagick6
```
Comment (dylanaraps) [inline community/emacs/depends]: Does the dependency list need updating too?

**Comment by a-schaefers** [comment]
i dont even want to maintain it. you can drop it

File: community/emacs/depends (hunk 1 above)
Comment (a-schaefers) [reply community/emacs/depends]: uhhhhhhh maybe

**Comment by dylanaraps** [comment]
Done :)

**Pull request merged**